### DISCUSSION TITLE
Why Client JDK2.0.4 Support DataType.String, Server Cant Load and Support

### DISCUSSION THREAD

**Discussion by zbhuang666** [milvus-io/milvus#16219] [Q&A and General discussion]
Why Client JDK2.0.4 Support DataType.String, Server Cant Load and Support

**Comment by yhmo** [comment]
String type is not yet supported in milvus server 2.0. The DataType.String is a reserved value in sdk side, including the python sdk and java sdk. In our plan the string type field will available in v2.1.

**Comment by zbhuang666** [comment]
您好，这边milvus2.0监控看的资源占比都很正常

但是collection进行load与relsease就卡死，服务不可用了，请求支援！
谢谢！




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "milvus-io/milvus"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年3月30日(星期三) 上午10:14
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [milvus-io/milvus] Why Client JDK2.0.4 Support DataType.String, Server Cant Load and Support (Discussion #16219)





 
String type is not yet supported in milvus server 2.0. The DataType.String is a reserved value in sdk side, including the python sdk and java sdk. In our plan the string type field will available in v2.1.

**Reply by yhmo** [reply]
collection load/release 卡住的问题比较复杂，你在issues里开一个issue，把collection的信息和卡住时的日志贴上。
